Sakura/Chat/chat.py

```
@@ -1,128 +1,122 @@
-        except Exception as e:
-            logger.error(f"❌ Failed to initialize chat client for a key: {e}")
-
-    if state.openrouter_clients:
-        logger.info(f"✅ {len(state.openrouter_clients)} chat client(s) (OpenRouter) initialized successfully")
-    else:
-        logger.error("❌ No chat clients could be initialized.")
+    logger.info("🫡 Initializing Gemini API key.")
+    try:
+        state.gemini_client = genai.Client(api_key=GEMINI_API_KEY)
+        logger.info("✅ Chat client (Gemini) initialized successfully")
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)